File: Ohtap E Commerce Domain Models/Development/NetCore/Models Core/Models Core/Communication/MessageList.cs
using Ohtap.ECommerce.Domain.Models.Core.System;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Core.Communication
{
    public class MessageList : Lista<Message>, IMessageList<Message>
    {
        public MessageList() : base()
        {

        }
        public void Add(string InfoCode, string Title, string Description = null)
        {
            base.Add(new Message(InfoCode, Title, Description));
        }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Core/Communication/MessageList.php
<?php
namespace Ohtap\ECommerce\DomainModels\Core\Communication;

use Ohtap\ECommerce\DomainModels\Core\System\Lista;

class MessageList extends Lista implements IMessageList
{
    public function AddMessage(string $InfoCode = null, string $Title = null, string $Description = null)
    {
        parent::Add(new Message($InfoCode, $Title, $Description));
    }
}
